React/Views/RCTView.m

```diff
@@ -341,6 +361,136 @@ - (NSString *)accessibilityValue
   }
   return nil;
 }
+#else // [TODO(macOS ISS#2323203)
+- (id)accessibilityValue {
+  if (self.accessibilityRole == NSAccessibilityCheckBoxRole) {
+    for (NSString *state in self.accessibilityState) {
+      id val = self.accessibilityState[state];
+      if (!val) {
+        continue;
+      }
+      if ([state isEqualToString:@"checked"]) {
+        if ([val isKindOfClass:[NSNumber class]]) {
+          return [val boolValue] ? @YES : @NO;
+        } else if ([val isKindOfClass:[NSString class]] && [val isEqualToString:@"mixed"]) {
+          return @(2);
+        }
+      }
+    }
+  }
+
+  // handle accessibilityValue
+
+  if (self.accessibilityValueInternal) {
+    id now = self.accessibilityValueInternal[@"now"];
+    id text = self.accessibilityValueInternal[@"text"];
```

Comment:
Same for all these strings but it'd probably be better to define them as consts once and ref those consts down here?